AsyncDisplayKit/ASDisplayNode.mm

```diff
@@ -155,7 +151,6 @@ - (void)_initializeInstance
   }
   _methodOverrides = overrides;
 
-  _flexBasis = ASRelativeDimensionUnconstrained;
```

Comment:
Are you confident this isn't needed?  I could be wrong, but I vaguely recall @nguyenhuy adding this after we encountered a certain bug.